_posts/2019-01-25-bill-israel.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title:  "PyTN 2019 Meet Our Organizers: Bill Israel"
+published: true
+categories: [blog, tech, conference]
+tags: [python, tech, conference, tennessee, nashville]
+---
+Bill Israel in the main organizers of PyTN. Whether you're speaking, sponsoring, or attending, he's been the one holding our ship together. Get to know him better and his opinions on the conference, Nashville, and more. He's a pretty rad guy.
+
+**What have you been working on lately that you're excited about? (It can totally be non-tech related!)**
```

Comment:
I realize it's a little late for this, but in the future, maybe we should use headings for these instead of just bolding the text? No need to change anything for this one, just a suggestion.

Reply:
For sure. I was adding Aliya and Josh anyway, so I went ahead and made the changes.